triplea_maps.yaml

```diff
@@ -1815,6 +1815,14 @@
     <br>
     Ultimate World Variants
     <br>Includes several Ultimate World Variants including FFAs
+- mapName: House of Habsburg
+  mapCategory: EXPERIMENTAL
+  url: https://github.com/triplea-maps/house_of_habsburg
+  version: 0.1
```

Comment:
This is just a version number used to indicate changes to the map (not the same as the map version) and it needs to just be an integer so "1" in this case.